src/libraries/System.Private.CoreLib/src/System/Double.cs

```diff
@@ -1562,7 +1562,7 @@ public static double Hypot(double x, double y)
                     else
                     {
                         // x or y is insignificant compared to the other
-                        result = x + y;
+                        result = ax + ay;
```

Comment:
The original C algorithm uses 'x' and 'y', but does so after having reassigned them to be the absolute value.

That is, it effectively does `x = abs(x), y = abs(y)`.

I double checked other usages of x/y and they are all fine/correct.